test/unit/layout/Panel.spec.js

```diff
@@ -3,10 +3,39 @@ import Panel from '../../../src/components/shared/layout/Panel'
 
 const Constructor = Vue.extend(Panel)
 
-const vm = new Constructor().$mount()
+// const vm = new Constructor().$mount()
+
+
+const vm = new Constructor({
+    propsData: {
+        title: 'Test Panel',
+        showCollapseToggle: true,
+    },
+}).$mount()
 
 describe('Panel', () => {
     it('should match the snapshot', () => {
         expect(vm.$el).toMatchSnapshot()
     })
 })
+
+describe('showCollapseToggle is false', () => {
+    beforeEach(() => {
+        vm.showCollapseToggle = false
+    })
+
+    it('shouldnt show the show / collapse button', () => {
+        expect(vm.$el.querySelectorAll('.circular .angle .icon').length).toBe(0)
```

Comment:
This isn't actually testing anything. 
`vm.$el.querySelectorAll('.circular .angle .icon')` will always be empty, as the selector is incorrect.

I will fix, look at my next commit for reference.